src/app/components/turnos/dashboard/update-contacto-direccion.component.ts

```diff
@@ -70,14 +71,17 @@ export class UpdateContactoDireccionComponent implements OnInit {
         this.tipoComunicacion = enumerados.getObjTipoComunicacion();
         this.tipoComunicacion.splice(this.tipoComunicacion.length - 1, 1);  // eliminamos 'Email'
 
-        if (this.paciente.contacto.length) {
+        if (this.paciente && this.paciente.contacto && this.paciente.contacto.length) {
             this.arrayContactos = this.paciente.contacto;
         } else {
             this.arrayContactos = [this.contacto];
         }
 
-        if (this.paciente.direccion[0].valor) {
+        if (this.paciente && this.paciente.direccion && this.paciente.direccion[0].valor) {
```

Comment:
```suggestion
        if (this.paciente && this.paciente.direccion && this.paciente.direccion.length && this.paciente.direccion[0].valor) {
```
No es necesario esto para evitar que pinche?